While runnng project on local system i'm getting "fatal error: 'cairo/cairo.h' file not found" Error

While running project on local system. i'm getting "fatal error: 'cairo/cairo.h' file not found" Error

i'm using flutter 3.10.6 version, fvm 2.4.1 and ubuntu 22.04

help me how i can run your project on locally


![image](https://github.com/ubuntu/app-center/assets/66154609/f0c84226-dd81-4134-8ecf-f9a26676aa7d)

I have install the toolchain  packages. see the below screen shot
![image](https://github.com/ubuntu/app-center/assets/66154609/b2ff7cd3-9603-44ba-9d5b-6bb1e18d9aa4)ow 

now I'm getting different error.
```
This will generate a JSON format file containing all messages that 
need to be translated.
Launching lib/main.dart on Linux in debug mode...
Building Linux application...                                   
/media/duggal/ssd_storage/programming/ubuntu/app-center/linux/flutter/ephemeral/.plugin_symlinks/handy_window/linux/handy_window_plugin.cc:9:10: fatal error: 'functional' file not found
/media/duggal/ssd_storage/programming/ubuntu/app-center/linux/flutter/ephemeral/.plugin_symlinks/url_launcher_linux/linux/url_launcher_plugin.cc:10:10: fatal error: 'cstring' file not found
/media/duggal/ssd_storage/programming/ubuntu/app-center/linux/flutter/ephemeral/.plugin_symlinks/screen_retriever/linux/screen_retriever_plugin.cc:7:10: fatal error: 'cstring' file not found
Exception: Build process failed
```
![image](https://github.com/ubuntu/app-center/assets/66154609/0bb8dd58-c887-482f-9fd6-c41a09f43cf0)

Now i'm getting below error. please help me how i can fix this error.
![image](https://github.com/ubuntu/app-center/assets/66154609/c203546d-44b8-443b-a674-96f10d9d10e2)
```"da": 69 untranslated message(s).
"fa": 81 untranslated message(s).
"fr": 16 untranslated message(s).
"hu": 79 untranslated message(s).
"id": 81 untranslated message(s).
"it": 68 untranslated message(s).
"ja": 69 untranslated message(s).
"ko": 79 untranslated message(s).
"pt": 73 untranslated message(s).
"tr": 88 untranslated message(s).
"uk": 69 untranslated message(s).
"zh": 70 untranslated message(s).
"zh_Hant": 68 untranslated message(s).
To see a detailed report, use the untranslated-messages-file 
option in the l10n.yaml file:
untranslated-messages-file: desiredFileName.txt
<other option>: <other selection> 


This will generate a JSON format file containing all messages that 
need to be translated.
Launching lib/main.dart on Linux in debug mode...
Building Linux application...                                   
CMake Error at /usr/share/cmake-3.22/Modules/CMakeTestCXXCompiler.cmake:62 (message):
  The C++ compiler

    "/usr/bin/clang++"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /media/duggal/ssd_storage/programming/ubuntu/app-center/build/linux/x64/debug/CMakeFiles/CMakeTmp

    Run Build Command(s):/usr/bin/ninja cmTC_93263 && [1/2] Building CXX object CMakeFiles/cmTC_93263.dir/testCXXCompiler.cxx.o
    [2/2] Linking CXX executable cmTC_93263
    FAILED: cmTC_93263 
    : && /usr/bin/clang++   CMakeFiles/cmTC_93263.dir/testCXXCompiler.cxx.o -o cmTC_93263   && :
    /usr/bin/ld: cannot find -lstdc++: No such file or directory
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    ninja: build stopped: subcommand failed.





  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:3 (project)


Exception: Unable to generate build files`

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -74,6 +74,11 @@ Translations are managed using [Weblate](https://hosted.weblate.org/projects/ubu
 
 Even though this repo currently consists of only a single package we provide a [Melos](https://docs.page/invertase/melos) configuration to make it straightforward to execute common tasks.
 
+Install Cairo:
+```
+sudo apt-get install libcairo2-dev
+```
+
 Install Melos:
 ```
 dart pub global activate melos

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -72,12 +72,9 @@ Translations are managed using [Weblate](https://hosted.weblate.org/projects/ubu
 
 [Install Flutter](https://flutter.dev/docs/get-started/install/linux) - the currently used version is specified in `.tool-versions`. If you're using [asdf](https://asdf-vm.com/) to manage your Flutter SDK, you can simply run `asdf install` to install the required version.
 
-Even though this repo currently consists of only a single package we provide a [Melos](https://docs.page/invertase/melos) configuration to make it straightforward to execute common tasks.
+Install the [Flutter Linux prerequisites](https://docs.flutter.dev/get-started/install/linux#linux-prerequisites)
 
-Install Cairo:
-```
-sudo apt-get install libcairo2-dev
-```
+We provide a [Melos](https://docs.page/invertase/melos) configuration to make it straightforward to execute common tasks.
 
 Install Melos:
 ```

